### DISCUSSION TITLE
should remappings.txt be ignored or upload to git repo?

### DISCUSSION THREAD

**Discussion by Azleal** [foundry-rs/foundry#9370] [Q&A] [answered]
I used to upload `remappings.txt` to git repo, however, when my teammate cloned the repo and `forge build` the project. it turned out that something wrong with `remappings.txt`. I'm use macOS while my teammate uses windows. so I wonder should I upload `remappings.txt` to git repo or should this file be ignored and generated with `forge remappings > remappings.txt`? thanks

**Comment by mattsse** [comment]
hmm, remappings.txt should work cross platform
could you perhaps share the remappings

**Comment by grandizzy** [comment]
bump @Azleal, per comment above, could you please provide a project sample to reproduce the issue? thank you

**Comment by grandizzy** [comment]
@Azleal going to convert this to discussion until more details to figure out if it's a bug or config related. Thank you!

**Reply by Azleal** [reply, marked as answer]
I apologize for the confusion earlier. After investigation, it appears this was a case of incorrectly copying libraries between projects. The issue occurred when my teammate, who is new to Foundry, attempted to import libraries from another project. I discovered this when trying to reproduce the issue.

Thank you @grandizzy @mattsse for your time and help on this.
